include/ListView/ListViewSubPanel.php

```diff
@@ -539,7 +539,9 @@ public function processListNavigation($smartyTemplateSection, $html_varName, $cu
             global $currentModule;
             global $app_strings;
 
-            $start_record = $current_offset + 1;
+            if( !isset($current_offset) || empty($current_offset) )                                                                    
```

Comment:
PSR-2: please use curly braces ( { ... }) when defining an if statement.

See our coding standards for details http://suitecrm.com/wiki/index.php/Coding_Standards

Minor formatting issue: please remove the spaces at the start and end of the brackets:

`if(!isset($current_offset) || empty($current_offset)) `